apm-agent-plugins/apm-ecs-logging-plugin/pom.xml

```diff
@@ -19,13 +19,19 @@
         <dependency>
             <groupId>co.elastic.logging</groupId>
             <artifactId>log4j-ecs-layout</artifactId>
-            <version>1.4.0</version>
+            <version>${version.ecs.logging}</version>
             <scope>provided</scope>
         </dependency>
         <dependency>
             <groupId>co.elastic.logging</groupId>
             <artifactId>log4j2-ecs-layout</artifactId>
-            <version>1.4.0</version>
+            <version>${version.ecs.logging}</version>
+            <scope>provided</scope>
+        </dependency>
+        <dependency>
+            <groupId>co.elastic.logging</groupId>
+            <artifactId>jul-ecs-formatter</artifactId>
+            <version>${version.ecs.logging}</version>
             <scope>provided</scope>
         </dependency>
```

Comment:
[for reviewer] the dependency on `jul-ecs-formatter` might only be required at runtime or for testing, and there is no compile-time dependency on it (excepted for tests), thus maybe `test` scope would be more appropriate.